src/frontend/src/eth/utils/qr-code.utils.ts

```diff
@@ -1,6 +1,10 @@
-import type { QrResponse, QrStatus } from '$lib/types/qr-code';
-import { decodePayment } from '@dfinity/ledger-icrc';
+import { erc20Tokens } from '$eth/derived/erc20.derived';
+import { enabledEthereumTokens } from '$eth/derived/tokens.derived';
+import type { EthereumNetwork } from '$eth/types/network';
+import { type QrResponse, type QrStatus } from '$lib/types/qr-code';
+import { decodeQrCodeUrn } from '$lib/utils/qr-code.utils';
 import { isNullish, nonNullish, type Token } from '@dfinity/utils';
+import { get } from 'svelte/store';
```

Comment:
If possible, please avoid using store imperatively in utils and rather pass the state or desired variable as parameter. That way the utils can be use dynamically.